docs/source/credentials.rst

```diff
@@ -2,39 +2,64 @@
 Credentials for smaht-submitr
 =============================
 
-Credentials can be placed your local file ``~/.smaht-keys.json`` which typically looks something like this::
+If you have been designated as a submitter for the project,
+and plan to use spreadsheet-based submission system (``smaht-submitr``)
+to view, upload, create, or update data from/to the SMaHT data portal,
+you are required to obtain and setup an access and secret key from the SMaHT data portal.
+These must stored in a specific format and file on your local system.
+Please follow these steps below to get your access keys and configure your local system for ``smath-submitr`` usage.
+
+Obtaining SMaHT Portal Access Keys
+----------------------------------
+
+#. Log in to the SMaHT `website <https://data.smaht.org>`_ with your username (email) and password.
+#. Once logged in, go to your **Profile** page by clicking **Account** on the upper right corner of the page.
+#. On your profile page, click the green **Add Access Key** button, and copy the **Access Key ID** and **Secret Access Key** values from the popup page. Note that *once the pop-up page disappears you will not be able to see the secret access key value*. However if you forget/lose your secret key you can always delete and add new access keys from your profile page at any time.
+#. Store these values the file ``~/.smaht-keys.json`` on your local machine; see the next section for details.
+
+Storing SMaHT Portal Access Keys
```

Comment:
I would consider removing references to staging even in this documentation - it is an internal implementation detail that is probably not necessary for outside users to know, and inside users other than the most advanced do not care much either, and requires further explanation ie: why the key is the same on data/staging